collector.go

```diff
@@ -28,6 +30,9 @@ var blockHeight = promauto.NewGauge(prometheus.GaugeOpts{
 var peerCount = promauto.NewGauge(prometheus.GaugeOpts{
 	Name: "peer_count", Help: "Number of peers connected"})
 
+var txPerSecond = promauto.NewGauge(prometheus.GaugeOpts{
```

Comment:
For MVP I always used Prometheus Gauge type for metrics. In case you didn't know, there are mulitple other types to describe metrics. 
Please check them out in case you didn't and check what matches best.